portal/apps/org.wso2.carbon.iam.user-portal.app/src/test/java/org/wso2/carbon/iam/userportal/actionobject/LoginPageAction.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.iam.userportal.actionobject;
+
+import org.openqa.selenium.WebDriver;
+import org.wso2.carbon.iam.userportal.pageobject.LoginPage;
+
+/**
+ * Action class for login.
+ */
+public class LoginPageAction {
```

Comment:
All the actions of a single page is written in a single action class so this class is LoginPageAction